tests/kernel/fatal/src/main.c

```diff
@@ -140,24 +140,17 @@ void blow_up_stack(void)
 
 void stack_sentinel_timer(void)
 {
-	u32_t cur_tick;
+        /* We need to guarantee that we receive an interrupt, so set a
+         * k_timer and spin until we die.  Spinning alone won't work
+         * on a tickless kernel.
+         */
+        struct k_timer timer;
 
-	/* Test that stack overflow check due to timer interrupt works */
 	blow_up_stack();
-	TC_PRINT("waiting for tick advance...\n");
-
-	/* This test has tickless kernel disabled, z_tick_get_32() returns
-	 * the current tick count without trying to offset it by checking
-	 * time elapsed in the driver since last update
-	 */
-	cur_tick = z_tick_get_32();
-
-	while (cur_tick == z_tick_get_32()) {
-		/* spin */
-	}
-
-	TC_ERROR("should never see this\n");
-	rv = TC_FAIL;
+        k_timer_init(&timer, NULL, NULL);
+        k_timer_start(&timer, 1, 0);
+        while(true)
```

Comment:
add parens

Reply:
I swear I thought there was some exception for empty blocks? :)

Regardless, need to correct whitespace anyway (been mucking with my emacs config and apparently broke the C style autodetect stuff)